src/changelog.rs

```diff
@@ -89,11 +90,21 @@ impl Changelog {
         Changelog { versions }
     }
 
-    pub fn to_markdown(&self, release: Option<&str>, print_authors: bool) -> String {
+    pub fn to_markdown(
+        &self,
+        release: Option<&str>,
+        print_authors: bool,
+    ) -> Result<String, String> {
```

Comment:
The overall PR seems ok to me :)

Just here we are used to see `Result<String, Error>` I guess.
Would you do the typed Error on this PR?

Reply:
Yep, will do or do you prefer another PR?

I just wanted to make it work and get your approval first.

Reply:
I prefer you do it in this PR then!